src/OpenTok/Util/Plugin/OpentokAuth.php

```diff
@@ -27,6 +29,19 @@ public function __construct($apiKey, $apiSecret)
     public function onBeforeSend(Event $event)
     {
         $request = $event['request'];
-        $request->addHeader('X-TB-PARTNER-AUTH', $this->apiKey.':'.$this->apiSecret);
+        $request->addHeader('X-OPENTOK-AUTH', $this->createAuthHeader());
+    }
+
+    private function createAuthHeader()
+    {
+      $token = array(
+          'ist' => 'project',
+          'iss' => $this->apiKey,
+          'iat' => time(),
+          'exp' => time()+(5 * 60),
+          'jti' => (float) rand() / (float) getrandmax(),
```

Comment:
the api-review actually specifies this as a string. could probably use http://php.net/manual/en/function.uniqid.php